docs/source/blueprints.rst

```diff
@@ -0,0 +1,42 @@
+Testing Blueprints with Titanoboa
+=====================================
+
+Titanoboa has first-class support for blueprints, a type of factory contracts described in `EIP 5202 <https://eips.ethereum.org/EIPS/eip-5202>`_.
+
+Blueprints provide an excellent way to implement factory patterns within Vyper, but be sure to keep track of verify blueprint addresses to ensure you are creating from the correct blueprint.
+
+Deploying Blueprints
+--------------------
+
+Any Vyper contract can be deployed as a blueprint, which allows clones of it to be deployed easily from Vyper contracts through `create_from_blueprint <https://docs.vyperlang.org/en/stable/built-in-functions.html?highlight=create_from_blueprint#create_from_blueprint>` builtin.
+
+In order to instantiate a blueprint simply use `boa.load_prtial` which is demonstrated in the below snippet to deploy a blueprint of the vyper contract `pair.vy`.
+
+.. code-block:: python
+
+    import boa
+    import pytest
+
+    @pytest.fixture(scope="session")
+    def blueprint(account):
+        with boa.env.prank(account):
+            pair = boa.load_partial('src/pair.vy')
+            return pair
+..
+
+This will return an instance of the blueprint which can then be passed into your Vyper contracts and used for blueprint factories. 
```

Comment:
this is a bit confusing i think - it returns a `VyperDeployer`, which can deploy as blueprint or as regular contract